lib/tasks/locale.rake

```diff
@@ -208,22 +194,24 @@ namespace :locale do
       "https://raw.githubusercontent.com/ManageIQ/react-ui-components/#{checkout_branch}/locale/react-ui-components.pot"
     ]
 
-    tmp_dir = Rails.root.join("locale/tmp").to_s
-    Dir.mkdir(tmp_dir, 0o700)
+    tmp_dir = Rails.root.join('locale/tmp').to_s
+    FileUtils.rm_rf(tmp_dir)
+    FileUtils.mkdir_p(tmp_dir)
```

Comment:
Minor, but I think you can just using the pathname directly.

```suggestion
    tmp_dir = Rails.root.join('locale/tmp')
    tmp_dir.rmtree
    tmp_dir.mkpath
```

and then way down below `tmp_dir.rmtree` at the end.

Reply:
You can also reuse the definition of the locale_die and the locale_tmp_dir down below by moving them up here.  Then you don't need this tmp_dor variable again.

Reply:
hmm, something at the time didn't support pathnames, hence the `to_s`... Maybe it was FileUtils itself.  If it was and I can use the pathnames directly, even better.  Good idea.  🤞

Reply:
Ah, it's calling system with the pathnames. That's easy enough to fix.

Reply:
done...